x-pack/legacy/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/create_analytics_form/job_description.tsx

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC } from 'react';
+import { EuiFormRow, EuiTextArea } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+const helpText = i18n.translate('xpack.ml.dataframe.analytics.create.jobDescription.helpText', {
+  defaultMessage: 'Optional descriptive text',
+});
+
+interface Props {
+  description: string;
+  setFormState: any; // TODO update type
```

Comment:
Is this TODO for a follow-up?

Reply:
Nope, I just missed it. Added the type here 56f779f84a0f7552df6a36cbf0d6105717a03a75